src/components/home/PanelResult.vue

```diff
@@ -203,40 +219,44 @@ export default Vue.extend({
   computed: {
     ...mapGetters({
       userGenes: 'getUserGenes',
-      userGenesInPanels: 'getUserGenesInPanels',
       institutionMap: 'getInstitutionMap',
+      panelSearchResult: 'getPanelSearchResult',
+      parsedGenes: 'getParsedGenes'
     }),
   },
+  watch: {
+    panelSearchResult: 'showPanelResult',
+  },
   methods: {
-    getPanelResult() {
-      return this.userGenesInPanels.map((panel: PanelSearchResult) => {
+    showPanelResult() {
+      this.notFoundGenes = this.parsedGenes.notFoundGenes
+      this.panelContent = this.panelSearchResult.map((panel: PanelSearchResult) => {
         const genesInPanel = panel.genesInPanel.map((gene: Gene) =>
-          gene.name.toUpperCase()
+            gene.name.toUpperCase()
         )
         const genesNotInPanel = panel.genesNotInPanel.map((gene: Gene) =>
-          gene.name.toUpperCase()
+            gene.name.toUpperCase()
         )
-
-        let institution = this.institutionMap.get(panel.name)
+        let institution = this.institutionMap.get(panel.name.toUpperCase())
         if (!institution) {
-          institution = {}
+          institution = {};
```

Comment:
Won't work with {{ item.institution.name }} online 101. Better to have it null

Reply:
If its null table is not being shown

Reply:
This is what I'm getting:
![image](https://user-images.githubusercontent.com/1919424/124791638-da030080-df11-11eb-9509-d2d8675ce4ec.png)

Reply:
That was an issue with my logic. I fixed it

Reply:
done